.circleci/config.yml

```diff
@@ -1,31 +1,53 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.0
+  plugin-ci: mattermost/plugin-ci@volatile
 
 workflows:
   version: 2
-  deploy:
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - master
     jobs:
       - plugin-ci/lint
       - plugin-ci/test
       - plugin-ci/build
-      - plugin-ci/deploy-release:
+  ci:
+    jobs:
+      - plugin-ci/lint:
           filters:
             tags:
               only: /^v.*/
+      - plugin-ci/coverage:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/build:
+          filters:
+            tags:
+              only: /^v.*/
+      - plugin-ci/deploy-ci:
+          filters:
             branches:
               only: master
+          context: plugin-ci
           requires:
             - plugin-ci/lint
-            - plugin-ci/test
+            - plugin-ci/coverage
             - plugin-ci/build
       - plugin-ci/deploy-release-github:
           filters:
             tags:
               only: /^v.*/
             branches:
-              only: master
+              ignore: /.*/
```

Comment:
why not `only: master`?

Reply:
Because with `only: master` we would deploy a release to github for every new commit on `master`, which isn't what we want, right?

Reply:
I think the tag filter above would prevent that. AFAIK, `ignore: /.*/` ignores all branches, which is the same as `only: master`, isn't it? (and we have `only: master` above). 0/5 on what we use, I just want to understand what's going on here a bit better.